build.sbt

```
@@ -47,7 +47,7 @@ libraryDependencies ++= {
 libraryDependencies ++= Seq(
   // testing
   "org.scalatest" %% "scalatest" % "3.2.13" % Test,
-  ("org.sangria-graphql" %% "sangria" % "3.2.0").cross(CrossVersion.for3Use2_13) % Test
+  ("org.sangria-graphql" %% "sangria" % "3.4.0").cross(CrossVersion.for3Use2_13) % Test
```

sangria 3.4.0 is now cross-compiled to scala 3
```suggestion
  "org.sangria-graphql" %% "sangria" % "3.4.0" % Test
```